elide-contrib/elide-test-helpers/src/main/java/com/yahoo/elide/contrib/testhelpers/graphql/GraphQLDSL.java

```diff
@@ -0,0 +1,535 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.testhelpers.graphql;
+
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.Argument;
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.Arguments;
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.Definition;
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.Document;
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.Edges;
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.Field;
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.Mutation;
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.Node;
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.Query;
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.Selection;
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.SelectionSet;
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.VariableDefinition;
+import com.yahoo.elide.contrib.testhelpers.graphql.elements.VariableDefinitions;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
```

Comment:
Is there a reason we need to use two different serialization frameworks (GSON and Jackson)?  Can we just use one of the two?